images/build/Dockerfiles/vso.bullseye.Dockerfile

```diff
@@ -0,0 +1,325 @@
+FROM mcr.microsoft.com/mirror/docker/library/buildpack-deps:bullseye AS main
```

Comment:
Any reason we don't base this off of our intermediate `oryx/githubrunners-buildpackdeps-bullseye` image?

Reply:
Yeah since this isn't in our approved images: https://eng.ms/docs/more/containers-secure-supply-chain/approved-images

Reply:
That shouldn't affect us since the intermediate images that we build locally are tagged with the prefix `oryxdevmcr.azurecr.io/private/oryx`, so this Dockerfile scan doesn't flag these images because it thinks they're later pushed to our ACR instance, but we just use that naming convention strictly to bypass the check.

Reply:
I ran into ` php: error while loading shared libraries: libxml2.so.2: cannot open shared object file: No such file or directory` which is similar to what I saw with `mcr.microsoft.com/mirror/docker/library/debian:bullseye` . Checking to see if I can make it work with this image